python/hail/docs/scans.rst

```diff
@@ -0,0 +1,78 @@
+.. _sec-scan:
+
+Scans
+===========
+
+The ``scan`` module is exposed as ``hl.scan``, e.g. ``hl.scan.sum``.
+
+The functions in this module perform rolling aggregations along the desired axis;
+For example, the ``count`` aggregator can be used as ``hl.scan.count`` to add an
+index along the rows of a `Table` or the rows or columns of a `MatrixTable`.
+
+For example, to add a rolling sum of a Table field, we could do:
```

Comment:
"For example, to compute a cumulative sum for a row field in a table:"

and make the field name `cum_sum`

"Note that the cumulative sum is exclusive of the current row's value."